src/routes/(app)/+page.server.ts

```diff
@@ -81,3 +86,46 @@ export const load: PageServerLoad = async ({ locals }) => {
     alert: await alert,
   };
 };
+
+// These actions will be accessed from throughout the whole application
+export const actions: Actions = {
+  // Delete single or multiple notifications on database
+  deleteNotification: async ({ locals, request }) => {
```

Comment:
I can't help but feel that noitifications don't really belong here. They don't really have anything to do with this page.

Reply:
Has been moved to separate page